### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.NullPointerException

### ISSUE THREAD

**Issue by cristianpetrache** [microsoft/azure-tools-for-java#4370]
IntelliJ build version: 2020.1.1 IC-201.7223.91
OS: Mac OS X
JDK: JetBrains s.r.o 11.0.6
Plugin version: 3.36.0-2020.1
Additional Info: None
Parent component: 
```
com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog$DialogRootPane[,0,22,830x475,invalid,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.NullPointerException
	at java.desktop/javax.swing.text.DefaultStyledDocument$ElementBuffer.create(DefaultStyledDocument.java:1469)
	at com.microsoft.intellij.runner.functions.deploy.ui.FunctionDeploymentPanel.fillFunctionApps(FunctionDeploymentPanel.java:141)
	at com.microsoft.intellij.runner.functions.deploy.ui.FunctionDeployViewPresenter.lambda$null$2(FunctionDeployViewPresenter.java:50)
	at com.microsoft.intellij.actions.functions.DeployFunctionAction.runConfiguration(DeployFunctionAction.java:81)
	at com.microsoft.intellij.actions.functions.DeployFunctionAction.lambda$onActionPerformed$0(DeployFunctionAction.java:62)
```

**Comment by Flanker32** [comment]
@cristianpetrache Thanks a lot for your report, could you please share a bit more infos like could this issue be stable reproduced?

**Comment by cristianpetrache** [comment]
It actually happened randomly. I was not even using the azure plugin at that time. I will try to pay more attention to what I am doing when the IDE signals the problem and come back with more data.

**Comment by cristianpetrache** [comment]
It might be related to the fact that at some time I tried to deploy one azure function project. I right clicked the module, selected Deploy > Deploy to azure functions and then I saw an error message: Failed to list function apps. As a result, the Function field in the ui remained empty. At that point I only managed to deploy with the maven plugin directly (mvn clean package azure-functions:deploy). I hope this helps!
